spotify_player/src/state/model.rs

```diff
@@ -155,6 +155,21 @@ pub struct Playlist {
     pub name: String,
     pub owner: (String, UserId<'static>),
     pub desc: String,
+    #[serde(default)]
+    pub is_folder: bool,
+    #[serde(default)]
+    pub level: (i32, i32), // current + target
+}
+
+#[derive(Deserialize, Debug, Clone)]
+/// A node to help building a playlist folder hierarchy
```

Comment:
can you add reference https://github.com/mikez/spotify-folders to explain the structure?